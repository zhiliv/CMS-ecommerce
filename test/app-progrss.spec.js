import { shallowMount } from '@vue/test-utils'
import Hr from '@/components/app/progress/progress.vue'

describe('app-progress.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(Hr, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple simple-xl',
        styles: 'background: white',
        id: 'indent',
        ariaValuenow: 50,
        ariaValuemin: 50,
        ariaValuemax: 50
      },
    })
  })
  it('Содержимое html для app-progress ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
