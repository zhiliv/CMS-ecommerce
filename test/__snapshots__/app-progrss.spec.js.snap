// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-progress.js Содержимое html для app-progress  1`] = `
<div
  class="progress"
  id="indent"
>
  <div
    aria-valuemax="50"
    aria-valuemin="50"
    aria-valuenow="50"
    class="progress-bar simple simple-xl"
    role="progressbar"
    style="background: white;"
  />
</div>
`;
